crates/turborepo-lib/src/config.rs

```diff
@@ -0,0 +1,132 @@
+use std::path::{Path, PathBuf};
+
+use anyhow::Result;
+use config::{Config, Environment};
+// Go's xdg implementation uses FOLDERID_LocalAppData for config home
+// https://github.com/adrg/xdg/blob/master/paths_windows.go#L28
+// Rust xdg implementations uses FOLDERID_RoamingAppData for config home
+// We use cache_dir so we can find the config dir that the Go code uses
+#[cfg(windows)]
+use dirs_next::cache_dir as config_dir;
```

Comment:
Shouldn't this be `data_local_dir`?
```suggestion
use dirs_next::data_local_dir as config_dir;
```

Reply:
Both resolve to the same path on Windows, but `data_local_dir` is a better name.